tests/console/sysstat.pm

```diff
@@ -66,10 +66,14 @@ sub run {
     validate_script_output "mpstat",     sub { /CPU    %usr   %nice    %sys %iowait    %irq   %soft  %steal  %guest  %gnice   %idle/ };
     validate_script_output "sar -u",     sub { /CPU     %user     %nice   %system   %iowait    %steal     %idle/ };
     validate_script_output "sar -n DEV", sub { /IFACE   rxpck\/s   txpck\/s    rxkB\/s    txkB\/s   rxcmp\/s   txcmp\/s  rxmcst\/s   %ifutil/ };
-    validate_script_output "sar -b",     sub { /tps      rtps      wtps   bread\/s   bwrtn\/s/ };
-    validate_script_output "sar -B",     sub { /pgpgin\/s pgpgout\/s   fault\/s  majflt\/s  pgfree\/s pgscank\/s pgscand\/s pgsteal\/s    %vmeff/ };
-    validate_script_output "sar -H",     sub { /kbhugfree kbhugused  %hugused/ };
-    validate_script_output "sar -S",     sub { /kbswpfree kbswpused  %swpused  kbswpcad   %swpcad/ };
+    if (is_opensuse) {
+        validate_script_output "sar -b", sub { /tps      rtps      wtps      dtps   bread\/s   bwrtn\/s   bdscd\/s/ };
+    } else {
+        validate_script_output "sar -b", sub { /tps      rtps      wtps   bread\/s   bwrtn\/s/ };
+    }
```

Comment:
I don't think it's good to distinguish on openSUSE as the output merely depends on the package version which is not openSUSE specific. Better turn around the condition and cover "all older versions" in an exception and newer, including Tumbleweed in "else":

```suggestion
    if (is_sle('<=15-SP2') || is_leap('<=15.2')) {
        validate_script_output "sar -b", sub { /tps      rtps      wtps   bread\/s   bwrtn\/s/ };
    }
    else {
        validate_script_output "sar -b", sub { /tps      rtps      wtps      dtps   bread\/s   bwrtn\/s   bdscd\/s/ };
    }
```

this assumes that – as is commonly done – any newer version than SLE 15-SP2 and Leap 15.2 correspondingly will have the updated version which is now in Tumbleweed.

Reply:
done